### PR TITLE
more pycodestyle fixes

### PR DESCRIPTION
**Pull request by allisonkarlitskaya**
The first one of these is pretty important: the test-static-code script, when run in a directory with nodes_modules/ checked out will try to scan all the python it finds in there, and reject an awful lot of it, which (if you have the git hook setup) will pretty much prevent you from ever pushing anything.

### CODE REVIEW

File: tools/test-static-code [hunk 1]
```diff
@@ -122,7 +122,7 @@ else
     # all normal files which are either:
     #   - filename is *.py
     #   - executable matching ^#!/usr/bin/python3
-    PYFILES="$(find . -type f '(' -name '*.py' -o -executable -execdir grep -q '^#!/usr/bin/python3' '{}' ';' ')'  -print)"
+    PYFILES="$(git grep -lI '^#!.*python3') $(git ls-files "*.py")"
```
Comment (martinpitt) [inline tools/test-static-code:126]: This ought to be guarded by `if [ -e .git ]` like in "valid JSON files"`, as we run test-static-code during `make check`. It is entirely plausible to build a released cockpit in a minimal build environment without `git` being installed.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Nice, thank you!

File: tools/test-static-code (hunk 1 above)
Comment (allisonkarlitskaya) [reply tools/test-static-code:126]: Good idea.  Thanks!

**Review by martinpitt — APPROVED** [review]
Cheers!

**Review by marusak — APPROVED** [review]
Than you!

**Pull request merged**